_lib/wordpress_event_processor.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import json
+import os.path
+import requests
+import dateutil.parser
+
+
+def posts_at_url(url):
+    """
+    Get WordPress posts as JSON from the given URL
```

Comment:
Ohhh... these are strings not comments, is this what's output to the console or something?